scrapers/python/logo-downloader.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+import requests
+import time
+
+mypath = 'logos'
+img_url = 'https://raw.githubusercontent.com/fuchsia-programming/fuchsia-programming.github.io/master/assets/images/logos/'
+
+if not os.path.isdir(mypath):
+   os.makedirs(mypath)
+
+
+def check_arguments():
+    if len(sys.argv) > 1:
+        return sys.argv[1]
+    return False
+
+
+def working_directory():
+    owd = os.getcwd()
+    os.chdir(owd + "/" + mypath)
+    cur_path = os.getcwd()
+
+    if cur_path:        
+        number = check_arguments()
+        if number == False:
```

Comment:
```suggestion
        if not number:
```

![codefactor](https://user-images.githubusercontent.com/418747/70678926-476f9080-1cdf-11ea-81c9-2f118f7e2f52.png)